src/apps/KintoAppRegistry.sol

```diff
@@ -107,7 +107,7 @@ contract KintoAppRegistry is
         address[] calldata appContracts,
         uint256[4] calldata appLimits
     ) external override {
-        require(appLimits.length == 4, "Invalid app limits");
+        require(childToParentContract[parentContract] == address(0), "Parent contract already registered as a child");
```

Comment:
hmm... but this will allow someone to prevent a protocol from registering...
1. Malicious user registers an app (0x123) with child 0xUniswap
2. Uniswap wants to register app (0xUniswap) but the require will be false and will revert.